src/connector/python/README.md

```diff
@@ -5,14 +5,16 @@
 
 ## Install
 
+You can use pip to install from github or pypi:
+
 ```sh
-git clone --depth 1 https://github.com/taosdata/TDengine.git
-pip install ./TDengine/src/connector/python
+pip install git+https://github.com/taosdata/taos-connector-python.git
+pip install taospy
```

Comment:
pip isntall url
or
pip install taospy

?

Reply:
`pip install url` not work for relative url for github projects, like  https://github.com/taosdata/TDengine/src/connector/python ,

Install with github project
```sh
pip install git+https://github.com/taosdata/taos-connector-python.git
pip install git@github.com:taosdata/taos-connector-python.git
```

Or install from pypi:

```sh
pip install taospy
```

Reply:
`pip install url` not work for relative url for github projects, like  https://github.com/taosdata/TDengine/src/connector/python ,

Install with github project
```sh
pip install git+https://github.com/taosdata/taos-connector-python.git
pip install git@github.com:taosdata/taos-connector-python.git
```

Or install from pypi:

```sh
pip install taospy
```

Reply:
I mean, we should use 'or' let the user know either way works well. Now it looks like we require users to execute two commands one by one.